go-controller/cmd/ovn-kube-util/app/nics-to-bridge.go

```diff
@@ -0,0 +1,134 @@
+package app
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/Sirupsen/logrus"
```

Comment:
logrus is just fine. Just seeking consistency. Pulling out glog in another PR then.